src/frontend/views/error.njk

```diff
@@ -0,0 +1,13 @@
+{% extends "common/layout/base.njk" %}
+{% set mainClasses = "govuk-main-wrapper--l" %}
+{% block pageTitle %}
+  Sorry, there is a problem with the service – Billings and Reconciliation – GOV.UK
```

Comment:
Should maybe be a separate ticket, but we should probably have a pageTitleFormatter() function or something that takes the main text you want to put in the title and adds a standard ending to it (like " -- Billings and Reconciliation -- GOV.UK").